src/Data/Morpheus/Types.hs

```diff
@@ -52,6 +54,7 @@ import           Data.Morpheus.Types.Internal.Resolving
                                                 , GQLRootResolver(..)
                                                 , Resolver(..)
                                                 , LiftEither(..)
+                                                , ActionGraphQL
```

Comment:
i think `WithOperation` would be more descriptive name for what this constraint Does. because of this bug https://github.com/morpheusgraphql/morpheus-graphql/issues/351. i am planing to rename `LiftEither` to `WithOperation`. and `liftEither` will only depend on `MonadTrans`.

Reply:
but i will do it in different PR. so for now it will be sufficient to rename `WithOperation`